content/blog/cumundi-guest-post/index.md

```diff
@@ -0,0 +1,314 @@
+---
+title: "Refactoring Pulumi Code with `aliases`"
+date: 2020-03-26
+meta_desc: "Cumundi uses Pulumi to build infrastructure components for their customers"
+meta_image: cumundi-pulumi.png
+authors:
+    - ringo-de-smet
+tags:
+    - Test Driven Developmnet
+---
+
+**Guest Article:** [Ringo De Smet](https://www.linkedin.com/in/ringodesmet/), Founder of [Cumundi](https://www.cumundi.cloud), standardizes on Pulumi for writing infrastructure as code libraries to deliver reusable building blocks. Pulumi enables him to iterate through the build-test-release cycle of these building blocks rapidly.
+
+Cumundi helps companies adopt the cloud more integrally. Companies are rebuilding the same 75% of the underlying infrastructure. But the shortage of people and time has led to limited investment in integrating all the non-functional requirements needed to run applications in the cloud.
```

Comment:
Replace with: "standardizes on Pulumi for writing infrastructure as reusable code libraries for his customers. Pulumi enables him to rapidly iterate through the build-test-release cycle of these building blocks.

Cumundi helps companies adopt cloud infrastructure in a more integral way. It has found that 75% of companies’ needs are covered by ‘vanilla’ infrastructure patterns. However, due to a shortage of people and time, there has been limited investment to take full advantage of cloud-native configurations - which can lead to inefficiency, poor performance, and higher costs."